src/types/vector3.ts

```diff
@@ -1,4 +1,17 @@
 export class Vector3 {
+    public static readonly AXIS_X= 0;
```

Comment:
Ensure your equals are equally spaced in all occurences and it's accepted. Thanks for the contribution :) 
```suggestion
    public static readonly AXIS_X = 0;
```